python/doc/theory/reliability_sensitivity/reliability_index.rst

```diff
@@ -1,44 +1,35 @@
 Reliability Index
 -----------------
 
-| The generalized reliability index :math:`\beta` is used under the
-  following context: :math:`\vect{X}` is a probabilistic input vector,
-  :math:`\pdf` its joint density probability, :math:`\vect{d}` a
-  deterministic vector, :math:`g(\vect{X}\,,\,\vect{d})` the limit state
-  function of the model,
-  :math:`\cD_f = \{\vect{X} \in \Rset^n \, / \, g(\vect{X}\,,\,\vect{d}) \le 0\}`
-  the event considered here and g(,) = 0 its boundary.
-| The probability content of the event :math:`\cD_f` is :math:`P_f`:
-
-  .. math::
+The generalized reliability index :math:`\beta_g` is used under the
+following context: let :math:`\vect{X}` be a probabilistic input
+vector with joint density probability  :math:`\pdf`, let
+:math:`\vect{d}` be a  deterministic vector, let :math:`g(\vect{X}\,,\,\vect{d})` be the limit state function of
+the model and let :math:`\cD_f = \{\vect{X} \in \Rset^n \,
+/ \, g(\vect{X}\,,\,\vect{d}) \le 0\}` be an event which probability
+:math:`P_f` is defined as:
+
+.. math::
     :label: PfX6
 
-      P_f = \int_{g(\vect{X}\,,\,\vect{d}) \le 0}  \pdf\, d\vect{x}.
+    P_f = \int_{g(\vect{X}\,,\,\vect{d}) \le 0}  \pdf\, d\vect{x}.
 
 The generalized reliability index is defined as:
 
 .. math::
 
     \beta_g = \Phi^{-1}(1-P_f) = -\Phi^{-1}(P_f).
 
-| As :math:`\beta_g` increases, :math:`P_f` decreases rapidly.
 
-These indices are available:
+As :math:`\beta_g` increases, :math:`P_f` decreases rapidly.
```

Comment:
```suggestion
As :math:`\beta_{gen}` increases, :math:`P_f` decreases rapidly.
```
Moreover, isn't the speed of convergence to zero as fast as the exponential function, depending on the opposite of the norm of the design point? In this case, I would write: "decreases (_exponentially_) rapidly" because this is a significant information here.

Reply:
ok!